gradle.properties

```diff
@@ -114,8 +114,8 @@ artifactRedirecting.androidx.compose.material3.adaptive.version=1.1.0-beta01
 artifactRedirecting.androidx.collection.version=1.5.0-beta01
 artifactRedirecting.androidx.annotation.version=1.9.1
```

Comment:
After the merge (you should push it, not squash it), please merge "Copy lifecycle from 16f8229b284" commit to "integration" discarding all changes (i.e. "empty" merge). This will resolve all potential conflicts.